psi4/src/core.cc

```diff
@@ -548,6 +548,12 @@ void py_psi_clean_options() {
     Process::environment.options.set_read_globals(true);
     read_options("", Process::environment.options, true);
     Process::environment.options.set_read_globals(false);
+    for (std::map<std::string, plugin_info>::iterator it=plugins.begin(); it!=plugins.end(); ++it) {
+        // Get the plugin options back into the global space
+        Process::environment.options.set_read_globals(true);
```

Comment:
Should move `Process::env` out of the loop? You may be bale to move this inside the `set_read_globals` two lines above, but I may be misunderstanding exactly what you are doing here.

Reply:
I treat both options and plugins as fragile snowflakes. What you say makes a lot of sense, but I'd want Jet/Andy's approval and testing on multiple failing plugins to believe it. :-)

Reply:
Should be safe to move the `Process:env...` to outside the loop.  Do you want to make the change before I approve?